documentation/manual/src/ornate/tutorial.md

```diff
@@ -629,7 +629,7 @@ In this section we have seen that deriving a Scala.js client works the same way
 client.
 
 We have seen that the endpoints library provides a minimal infrastructure that is designed to
-be extended according to application-specific needs. In our case we saw how the `OptionalResponses`
+be extended according to application-specific needs. In our case we saw how the `OptionalResponses` //TODO OptionalResponses are no longer there
```

Comment:
Oops… we have to find another example… Maybe `Assets` or `AuthenticatedEndpoints`?